patches/api/0351-FoodLevelChangeEvent-Expose-Saturation.patch

```diff
@@ -0,0 +1,67 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Sat, 25 Dec 2021 23:04:06 -0500
+Subject: [PATCH] FoodLevelChangeEvent Expose Saturation
+
+
+diff --git a/src/main/java/org/bukkit/event/entity/FoodLevelChangeEvent.java b/src/main/java/org/bukkit/event/entity/FoodLevelChangeEvent.java
+index 7f4c5ea8607b08f8676528cc71b0b312575809ed..b1f5ed6656d9d80a3dcca2a5bacf9fc8079ab321 100644
+--- a/src/main/java/org/bukkit/event/entity/FoodLevelChangeEvent.java
++++ b/src/main/java/org/bukkit/event/entity/FoodLevelChangeEvent.java
+@@ -15,16 +15,27 @@ public class FoodLevelChangeEvent extends EntityEvent implements Cancellable {
+     private boolean cancel = false;
+     private int level;
+     private final ItemStack item;
++    private float saturationModifier; // Paper - Expose Saturation
+ 
+     public FoodLevelChangeEvent(@NotNull final HumanEntity what, final int level) {
+         this(what, level, null);
+     }
+ 
++    @Deprecated // Paper - Expose Saturation
+     public FoodLevelChangeEvent(@NotNull final HumanEntity what, final int level, @Nullable final ItemStack item) {
+         super(what);
+         this.level = level;
+         this.item = item;
++        this.saturationModifier = what.getSaturation(); // Just incase, make sure the player's saturation doesn't change. // Paper - Expose Saturation
```

Comment:
This doesn't support what I would think most people take `saturationModifier` to mean, that its the change in saturation. I would call it `saturationLevel` if you want it to be what the player's saturation is set to after, not changed to. **But** based on what most of the calls to this event do, I think `saturationDelta` or `saturationChange` might be better, and if thats the case setting it to the entities sat level is incorrect

Reply:
Ah, yea... I think I defo misunderstood what this value actually meant. Thanks so much for this, I'll make the given changes.